### ISSUE TITLE
"Red" sentences can now be translated. They didn't used to be.

### ISSUE THREAD

**Issue by ckjpn** [Tatoeba/tatoeba2#1013]
I changed this to -1 yesterday, but someone has translated it since then.
I'm not sure when this quit working, but it used to be that if a sentence's "correctness level" was -1, members could not add translations.

https://tatoeba.org/eng/sentences/show/4927737

![screen shot 2016-02-24 at 02 16 43](https://cloud.githubusercontent.com/assets/7934273/13260020/1756aeca-da9d-11e5-9a6c-f098b240dee2.png)

**Comment by jiru** [comment]
Looking at [unapproved sentences](https://tatoeba.org/eng/sentences/search?query=&from=und&to=und&orphans=no&unapproved=yes&user=&tags=&has_audio=&trans_filter=limit&trans_to=und&trans_link=&trans_user=&trans_orphan=&trans_unapproved=&trans_has_audio=&sort=words), I still can’t translate them.

As far as I know, the translation restriction is only performed in the view but not in the controller, so it’s quite easy to bypass, either on purpose, or by leaving the page open (not reloading it) while the correcteness is changed (that’s the most probable scenario I can think of).

**Comment by ckjpn** [comment]
I'm pretty sure it was several hours later that the translation was added.

It's hard to tell exactly, but I think https://tatoeba.org/eng/user/profile/r381318 may have even registered after I changed the sentence to -1.

I think I set it to -1 when I left my comment.

If it's a bug that can't be duplicated, though, I think you can close this issue.

**Comment by jiru** [comment]
People can leave web pages opened for days by putting their computer to sleep or hibernation.

**Comment by jiru** [comment]
Or you might as well directly ask the translation author.

**Comment by jiru** [comment]
> It's hard to tell exactly, but I think https://tatoeba.org/eng/user/profile/r381318 may have even registered after I changed the sentence to -1.

I checked the dates from the database and you’re right. So my guess about users leaving pages open was false. So I searched a bit and found that it’s simply possible to translate unapproved sentences by clicking on their translate button (regardless of what the tooltip says), giving the page includes at least one non unapproved sentence. It’s very likely what actually happened.
